src/ft2font_wrapper.cpp

```diff
@@ -497,12 +515,13 @@ static PyObject *PyFT2Font_get_kerning(PyFT2Font *self, PyObject *args, PyObject
 {
     FT_UInt left, right, mode;
     int result;
+    int fallback = 1;
```

Comment:
it can theoretically, this parameter is like a flag to turn fallback off or on.. for now it is on by default, parsing from Python/documenting API change could suit another PR